src/components/App/App.js

```diff
@@ -16,12 +16,13 @@ class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      userName: null
+      userName: null,
+      accountType: null
     }
   }
 
-  handleSubmit = (name) => {
-    this.setState({userName: name})
+  handleSubmit = (name, accountType) => {
```

Comment:
This could have been handled without the need for multiple params by just passing up this.state or [...this.state]